build.sbt

```diff
@@ -30,10 +31,11 @@ val allVersionsOption = Seq(
   "-feature", // Emit warning and location for usages of features that should be imported explicitly.
   "-language:existentials", // Existential types (besides wildcard types) can be written and inferred
   "-language:higherKinds", // Allow higher-kinded types
-  "-language:implicitConversions", // Allow definition of implicit functions called views
+  "-language:implicitConversions" // Allow definition of implicit functions called views
+)
 
+val scala2allVersionsOption = Seq(
```

Comment:
Not all options in `allVersionsOption` were applicable in scala3 so I extracted the ones that's not applicable in scala3 to `scala2allVersionsOption`